DataHub/Operation/04_ManageDashboards.md

```diff
@@ -0,0 +1,221 @@
+
+No prerequisites to fulfill.
+
+#### Procedure
+
+*DataHub > Overview > Tab DASHBOARDS*
+
+![Dashboards](../../Assets/Screenshots/DataHub/Overview/Dashboards.png "[Dashboards]")
+
+1. Click the ![Add](../../Assets/Icons/Plus06.png "[Add]") (Add) button in the header.   
+  The *Add a new tab* window is displayed. By default, the *Create new tab* tab is preselected.
+
+  ![Add a new tab](../../Assets/Screenshots/DataHub/Overview/AddNewTab.png "[Add a new tab]")
+
+2. Enter a name for the dashboard tab in the *Tab name* field.
+
+3. Click the [SAVE] button in the bottom right corner.   
+  The *Add a new tab* window is closed. The new dashboard has been saved and is displayed.
+
+  > [Info] You can create an unlimited number of dashboard tabs and customize them to the desired purpose.
+
+
+
+## Customize a dashboard
+
+Customize a dashboard to your special needs by adding the desired dashlets, deleting unnecessary dashlets, select the viewport and define the dashlets arrangement.
+
+### Add a dashlet
+
+Add the desired dashlets to your dashboard to get an overview over the processes and entities within the *DataHub* module.   
+Depending on the purpose of the dashboard, you can add different sets of dashlets to different dashboards.
+
+#### Prerequisites
+
+A dashboard tab has been created, see [Create a dashboard](#create-a-dashboard).
+
+#### Procedure
+
+*DataHub > Overview > Tab DASHBOARDS*
+
+![NewTab](../../Assets/Screenshots/DataHub/Overview/NewTab.png "[NewTab]")
+
+1. Select the tab of the dashboard to which you want to add a dashlet.    
+  The selected dashboard tab is displayed.
+
+2. Click the ![Settings](../../Assets/Icons/Settings01.png "[Tool]") (Settings) button in the upper right corner.    
+  The *Edit tab* view is displayed.
+
+  ![Tab](../../Assets/Screenshots/DataHub/Overview/Tab.png "[Tab]")
+
+3. Click the [ADD ELEMENT] button in the upper right corner.    
+  The *Dashlet picker* window is displayed.
+
+  ![Dashlet picker](../../Assets/Screenshots/DataHub/Overview/DashletPicker.png "[Dashlet picker]")
+
+4. Select the checkboxes of the dashlets you want to add to your dashboard. If you click the checkbox in the header, all dashlets in the list are selected.
+
+  > [Info] For detailed information about the available dashlets, see [Dashlet picker](../UserInterface/01a_Dashboards.md#dashlet-picker).
+
+5. Click the [ACCEPT] button in the bottom right corner of the window.   
+  The selected dashlets have been added to the dashboard. The *Dashlet picker* window is closed.
+
+6. Click the [SAVE] button.   
+  The dashboard has been saved. The *Edit tab* view is closed.
+
+  > [Info] If you want to change the dashlet positioning, see [Change the dashlets arrangement](#change-the-dashlets-arrangement).
+
+
+
+### Change the viewport
+
+Change the viewport to adjust the view of the dashboard to the currently used device.
+The viewport is saved for a single dashboard.
+So you can create dashboards with different viewports.
```

Comment:
Ich würde beide Sätze verbinden: The viewport is saved for a single dashboard, so you can create dashboards with different viewports.